### DISCUSSION TITLE
About "recompiled the package"  operations

### DISCUSSION THREAD

**Discussion by IlllIIIlllIl** [SynxFlow/SynxFlow#19] [Q&A] [answered]
I have already add code control equations to the synxflow/apps/cudaDebrisFlowSolverPybind/cuda_debris_flow_solver. Cu, I updated the code in synxflow/lib include and src's.h and.cu. What should I do next is package the synxflow file into python\Lib\site-packages\synxflow? It's like you said "recompiled the package"?And how to do this recompilation step,does this step have a professional name, and what software or method should I use to implement it.
![46789811486aba08b6ce1b955cad499](https://github.com/user-attachments/assets/6e9ef0f6-c13c-4c32-8e69-b0f99d0f794e)
![52fe38d78bf11b65c68cffb24df51e9](https://github.com/user-attachments/assets/1b45c96c-b2c6-4a0c-874e-9387540ee7f6)
Is the specific thing I need to do to recompile the synxflow file in Figure 1 into a file format like the landslide.cp310-win_amd64 in Figure 2?

**Comment by xiaxilin** [comment, marked as answer]
Hi, the easiest way to do this is to run 'python setup.py install', after which the package will be installed again.
